exotica_examples/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ find_package(catkin REQUIRED COMPONENTS
   exotica_aico_solver
   exotica_core_task_maps
   exotica_ik_solver
+  exotica_scipy_solver
```

Comment:
It's not compiling, so not necessary here.

Reply:
Reverted `CMakeLists.txt`, squashed, and force pushed.